miden-lib/asm/scripts/ASWAP.masm

```diff
@@ -0,0 +1,53 @@
+use.miden::sat::account
+use.miden::sat::note
+use.miden::sat::tx
```

Comment:
I think I would use the wallet procedures to receive and send the asset for simplicity. `miden-lib/asm/miden/wallets`. Sending the asset using the wallet create a Note

Reply:
Yes